### ISSUE TITLE
[BUG] Overrides “Open in Cydia”

### ISSUE THREAD

**Issue by CySxL** [AppTapp/Installer-Issues#11]
**Describe the bug**
Can"t open repo addresses with cydia, the "open with cydia" button has been hijacked by installer5, which didn’t happen before.

**To Reproduce**
click a repo and boom

**Does the issue always occur when trying to reproduce it?** 
Yes

**Expected behavior**
Let me open with cydia normally.

**Screenshots or videos**
If applicable, add screenshots or videos to help explain your problem.

**Device (please complete the following information):**
 - Installer version: [

 5.0RC6-1]
 - iOS: [ iOS 12.4]
 - Jailbreak [unc0ver 3.7.0~b4]
 - Device [ iPhone 6s]

**Additional context**

**Comment by AppTapp** [comment]
This is because if people do not have Cydia installed, they would want to open any repository or tweak in Installer (since Installer would be installed in this case). For some reason iOS prioritises our URL scheme to open cydia:// links over Cydia itself. 

We'll add an option for this in Settings.

**Comment by CySxL** [comment]
Does the option already exist ?
I think i didn’t find it lol😂

**Comment by blanxd** [comment]
I was investigating this some time ago (some discussed about it on reddit), was thinking about making some tweak for choosing which of your pkg managers would open these links, but it seemed the scheme gets registered fifo, so the last pkg manager intalled wins. At least in my case, re-installing Cydia made it open the links again, the same with any pkg manager. (would have needed to modify each manager's plist and run uicache etc, didn't want to go messing with things like that so abandoned the idea. a simple forced reinstallation of Cydia takes it back)

**Comment by CreatureSurvive** [comment]
@blanxd I looked into this as well and it looks like MobileCoreServices Registers these url schemes during application install, and iOS does not offer any kind of native or private api to override this behavior. The only means I’ve determined that can resolve this is either a tweak to manage open in support, or removing the scheme support entirely

**Comment by cottontails2** [comment]
> We'll add an option for this in Settings.

I made a topic similar to this, i would love to see a solution, you guys said you'd add an option about 3 months ago. I get this stuff can take a long time to correct so in no means am i trying to sound pushy or impatient, but i would like to know if this is something you guys are still working on or not? I think part of the problem is the login button that is in some repositories, but is missing in others such as the Chariz repo, if we could login through the repository in Installer 5 and verify our purchases that could be a partial solution. I've already contacted someone from the Chariz repo and they say they're unfamiliar with Installer 5, they say purchasing their tweaks work perfectly fine on Cydia, Zebra and Sileo. So i am unsure if the issue is on their end or if it is a problem with the Installer 5 app, they seem unsure to, but the guy i spoke to forwarded to another member and they're going to investigate it. Maybe you guys can contact them and work together on a solution they're email is support@hbang.ws

Thanks for taking the time to read this.
